bower.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "polymer": "Polymer/polymer#^1.9.0",
     "iron-image": "^2.1.2",
-    "paper-tooltip": "^2.0.1",
+    "kwc-paper-tooltip": "git@github.com:KanoComponents/kwc-paper-tooltip#b8ba64d13701952a8d8d7a82e43327a3afd4f5c5",
```

Comment:
Do you think we could perhaps [make a release](https://github.com/KanoComponents/kwc-paper-tooltip/releases) and refer to the `kwc-paper-tooltip` version?